src/styles/ProjectImageBox.css

```diff
@@ -0,0 +1,8 @@
+.projectImageBox {
+    display: flex;
+    justify-content: center;
+}
+.projectImageBox img {
+    border-radius: 4px;;
```

Comment:
This also has two ;; in it, cause the image to not have a border radius

Reply:
Sorry, I'm not very used to Git yet. Does this mean that you deleted the extra semicolon and I can just merge this, or should I fix it and push the fix again?

Reply:
ah no, a review just means that I put comments on certain section of your changes, I don't make the changes myself. It's more of a "hey this looks incorrect, can you fix it and push again" kinda thing ^^

Reply:
I can't see that in my code... it won't let me commit since I haven't made any changes.

Reply:
I actually noticed that too when I cloned so I dunno what happened >.<

Reply:
should I just merge it then?

Reply:
Maybe you should remove the static image and profile container first, so that it becomes fully correct?